app/talk/inbox-conversation.cjsx

```diff
@@ -67,6 +67,11 @@ module?.exports = React.createClass
       .then (message) =>
         @setConversation()
 
+  handleDelete: (e) ->
+    e.preventDefault()
```

Comment:
Looks great, only request is to add a window.confirm to the handler since it's a harsh action